platform/commonUI/browse/src/PaneController.js

```diff
@@ -31,12 +31,20 @@ define(
          * @constructor
          * @memberof platform/commonUI/browse
          */
-        function PaneController($scope, agentService, $window) {
+        function PaneController($scope, agentService, $window, $location, $attrs) {
             var self = this;
             this.agentService = agentService;
 
-            // Fast and cheap: if this has been opened in a new window, hide panes by default
-            this.state = !$window.opener;
+            var alias = $attrs.alias;
+            var aliasParams = $location.search()["hide" + alias];
```

Comment:
This should be conditional, then you don't need comments and it's more clear what it's doing.

Also, a hasOwnProperty check will check if the parameter is defined, instead of parsing the truthiness of it, which could be confusing.

e.g.

```javascript
if ($attrs.hideParameter && location.search().hasOwnProperty($attrs.hideParameter)) { 
    this.state = false; 
    $location.search(hideParam, undefined);
} else { 
    this.state = true; 
}
```